### PR TITLE
{data}[system/system] AMAPVox v1.9.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node109.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/3e8e1fadf6a91aec5c6d560aae14de54 for a full test report.

File: easybuild/easyconfigs/a/AMAPVox/AMAPVox-1.9.4-Java-11.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+source_urls = ['http://amap-dev.cirad.fr/attachments/download/1916/']
+sources = ['AMAPVox-%(version)s.zip']
+checksums = ['6d3e65fbd17a8e3a45962f5d1ce60a58ad0bfd899fa704dc172f2819fc309b74']
+
+dependencies = [('Java', '11')]
+
+postinstallcmds = ["chmod a+x %(installdir)s/AMAPVox.sh"]
+
+sanity_check_paths = {
+    'files': ['AMAPVox.sh', 'AMAPVox-1.9.4.jar'],
```
Comment (jfgrimm) [inline easybuild/easyconfigs/a/AMAPVox/AMAPVox-1.9.4-Java-11.eb:21]:
```suggestion
    'files': ['AMAPVox.sh', 'AMAPVox-%(version)s.jar'],
```

File: easybuild/easyconfigs/a/AMAPVox/AMAPVox-1.9.4-Java-11.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+version = '1.9.4'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://amap-dev.cirad.fr/projects/amapvox'
+description = "LiDAR data voxelisation software"
+
+toolchain = SYSTEM
+
+source_urls = ['http://amap-dev.cirad.fr/attachments/download/1916/']
+sources = ['AMAPVox-%(version)s.zip']
```
Comment (jfgrimm) [inline easybuild/easyconfigs/a/AMAPVox/AMAPVox-1.9.4-Java-11.eb:13]:
```suggestion
sources = [SOURCE_ZIP]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node151.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b558b39ab94cb672a878e5a3a18ed2e5 for a full test report.

**Review by jfgrimm — APPROVED** [review]
LGTM

**Comment by jfgrimm** [comment]
Going in, thanks @boegel!

**Pull request merged**